high_card.rb

```diff
@@ -1,5 +1,28 @@
 # Use these two arrays to generate a deck of cards.
-ranks = ["A", 2, 3, 4, 5, 6, 7, 8, 9, 10, "J", "Q", "K" ]
+ranks = ["A", "K", "Q", "J", 10, 9, 8, 7, 6, 5, 4, 3, 2]
 suits = [ "hearts", "spades", "clubs", "diamonds" ]
 
 players = []
+
+deck = ranks.product(suits).shuffle
+
+puts players.length.to_s + " players so far. Enter a player name, or type 'play':"
+
+until  players.include? "play" do
```

Comment:
this feels a little smelly. "play" should never really enter the array, it's explicitly for players. Instead of removing it after the fact, don't allow it to beginning with. Maybe something like :
```ruby
user_input = nil
while user_input != "play"
  user_input = gets.chomp
  players << user_input
  puts players.length.to_s + " players so far. Enter a player name, or type 'play':"
end